src/common/mint-utils.ts

```diff
@@ -61,11 +61,13 @@ export function createIpfsMetadata(
   datasetTitle: string,
   datasetDescription: string,
   dataNFTStreamPreviewUrl: string,
-  address: string
+  address: string,
+  extraAssets: string[]
```

Comment:
Only add extraAssets to the NFT Metadata IF it's provided by the user, i.e. as it's optional and cal be empty... if it's empty we should not add it to NFT metadata file.

I.e. do an : if (extraAssets && extraAssets.length > 0) check.

@damienen

Reply:
Solved.